src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -197,6 +206,35 @@ private void seekToOffsets(List<TopicPartition> partitionIds) {
     return Collections.emptyList();
   }
 
+  @Override
+  public void commit() {
+    setCommitTime();
```

Comment:
The setter seems redundant - just use `commitTime = time.milliseconds();`?

Reply:
Maybe rename to `successfulCommitTime` to clarify that this is only set after the full offset commit succeeds.